rosdep/python.yaml

```
@@ -7863,6 +7863,10 @@ python3-pyside2.qtuitools:
     '*': [python3-pyside2.qtuitools]
     bionic: null
     xenial: null
+python3-pysnmp:
+  debian: [python3-pysnmp4]
+  gentoo: [dev-python/pysnmp]
+  ubuntu: [python3-pysnmp4]
```

```suggestion
python3-pysnmp:
  debian: [python3-pysnmp4]
  fedora: [python3-pysnmp]
  gentoo: [dev-python/pysnmp]
  rhel: [python3-pysnmp]
  opensuse: [python3-pysnmp]
  ubuntu: [python3-pysnmp4]
```

Add suggestions from the repo check tool.